Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -140,6 +144,13 @@ class BrowserViewController: UIViewController {
         self.readerModeCache = ReaderMode.cache(for: tabManager.selectedTab)
         self.crashedLastSession = crashedLastSession
         self.safeBrowsing = safeBrowsingManager
+        
+        #if NO_REWARDS
+        rewards = nil
```

Comment:
Redundant?

Reply:
it is actually needed since `rewards` is a let constant